scripts/install_libvirt_deps.sh

```diff
@@ -1,34 +1,7 @@
 #!/bin/bash
 
-if [ $# -lt 1 ]; then
-  echo "Usage $0 <lp-path>"
-  echo
-  echo lp-path: path to linchpin source
-  exit 1
-fi
+dnf install libvirt-devel -yq
```

Comment:
Removing the test for the root user is problematic. It requires that the user know that they need to run this script as root. It doesn't do the necessary to determine the user for them, which might prompt for a password via the sudo settings. The old script didn't assume the user would be root. Especially since the pip install below this would need to be for the virtualenv, whcih would likely be an unprivileged user.